markdownhighlighter.cpp

```diff
@@ -2051,7 +2026,7 @@ void MarkdownHighlighter::highlightInlineRules(const QString &text) {
 */
 int MarkdownHighlighter::highlightInlineSpans(const QString &text,
                                               int currentPos, const QChar c) {
-    //clear code span ranges for this block
+    // clear code span ranges for this block
```

Comment:
Nothing changed below this spot, except the formatting.  The next change is in the next file